intermediate-exercises/src/components/01-LifeCycle.js

```diff
@@ -11,34 +11,37 @@ import React from 'react';
 class LifeCycle extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { name: 'Pesto' };
+    this.state = { name: "Pesto" };
+    this.updateState = this.updateState.bind(this);
   }
 
   // This code will be called when the component finishes mounting
   // (so it is visible for a user).
   componentDidMount() {
-    // Task 1: Display a message "I'm mounted!" in developer's console when the
-    //         component finishes mounting.
-    //         Use `console.log` function for it.
+    console.log("I'm mounted");
   }
 
   componentDidUpdate(prevProps, prevState) {
-    // Task 2: Display a message "Updated!" in developer's console
-    //         when the component updates.
-    //         Here you also need to use the console.log function.
-    //         Notice that in this lifecycle method you have an access
-    //         to previous values of properties and state.
+    console.log("Updated!");
```

Comment:
You should check for previousState here if the prevState name is the same as the new state then this should not run otherwise log to the console updated !